### ISSUE TITLE
Need help inlining svg with webpack javascript

### ISSUE THREAD

**Issue by hiousi** [JetBrains/svg-mixer#57]
This is not an issue, but did not find any other place to ask. 

I'm using extract-svg-sprite-webpack-plugin and import all svg I need:
```javascript
import user from './svg/user.svg';
import menu './svg/menu.svg';
```

With the help of plugin.js a sprite.svg file is created  on the filesystem.
That great, but need more for my use case

I don't really need to emit a file. I would like to be able to inject the sprite content to the html DOM with js.
something like this:

```javascript
import user from './svg/user.svg';
import menu './svg/menu.svg';

var content = 'SVG Content Goes Here';
document.addEventListener('DOMContentLoaded', function(){
     document.body.prepend(content);
});
```

I saw react samples in the repo. As I do not use react I looking for a way of doing without it.

**Comment by kisenka** [comment]
Check this test case https://github.com/JetBrains/svg-mixer/tree/master/packages/extract-svg-sprite-webpack-plugin/test/cases/html-webpack-plugin-inline-sprite

**Comment by hiousi** [comment]
@kisenka not sure that the case is the same as mine.  If a understand well, html-webpack-plugin will produce a static html page with svg already in it. But what I'm looking for is to inject the svg XML code on page's body after the page is loaded (with a javascript script that the page in the browser can load and cache).

**Comment by kisenka** [comment]
Ah, ok. You can try svg-sprite-loader for your case

**Comment by hiousi** [comment]
yes I'm coming from there. This example works: https://github.com/kisenka/svg-sprite-loader/tree/master/examples/browser-sprite ,  since svg-mixer was newer I thought it's better to use it. 

Never mind, I ended with a far simpler solution, with less javascript in the browser.
Let extract-svg-sprite-webpack-plugin and svgo-loader do their work and emit a sprite.svg file.
Load a symbol from the file with `<svg><use href="sprite.svg#symbol_id"/></svg>`. 
Added https://github.com/Keyamoon/svgxuse polyfill for browsers that fails to use external sprite.

Now I have a minor issue with webpack-manifest-plugin, I think it's same as this one : https://github.com/kisenka/svg-sprite-loader/issues/166 . Tell me if it's usefull to open an issue here.

thanks.
